<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentSettings</class>
 <widget class="QWidget" name="ParentSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>731</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="basicSettings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Basic Settings</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="AdvancedSettingsTab">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Advanced Settings</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>760</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>760</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
